### DISCUSSION TITLE
Reverse trend in discharge capacity during a battery discharge & charge

### DISCUSSION THREAD

**Discussion by Ramesh-Isukapalli** [pybamm-team/PyBaMM#3503] [Q&A]
After running a simulation accounting for all degradation mechanisms, I have collected data and could see the reverse trend followed by discharge capacity during charge and discharge. The discharge capacity is increasing instead of decreasing during discharge and decreasing instead of increasing during charge. what could be the reason?

**Comment by fsenoner** [comment]
Hi Ramesh,

maybe this post could help you: https://github.com/pybamm-team/PyBaMM/discussions/2472

Regards

**Reply by Ramesh-Isukapalli** [reply]
Hi fsenoner,

Thanks for the reply.
My question is about discharge capacity variable which is being tracked at cycle and over the period of a cycle during the discharge period the discharge capacity is increasing from value around 0 to the nominal rated capacity.

The post #2472 is talking about capacity summary variable which is different.

Regards,

**Comment by Akila1993** [comment]
Hi  @Ramesh-Isukapalli and All, 

Did you get a solution for this? I too have the same issue.  

@tinosulzer @brosaplanella could you please help us with this?

Thank you.

**Reply by Ramesh-Isukapalli** [reply]
Hi @Akila1993, 

I did not any get answer for that so far. However, I have convinced myself that may be I was interpreting it wrong so instead I considered subtracting these values of discharge capacity which is then giving me typical discharge capacity behaviour. I'm not sure whether it is correct or not

**Reply by Akila1993** [reply]
@Ramesh-Isukapalli you may refer this [discussion](https://github.com/pybamm-team/PyBaMM/discussions/3685#discussion-6026473).

**Reply by Ramesh-Isukapalli** [reply]
Hi @Akila1993 ,

So is that mean we should calculate SOC by subtracting the discharge capacity value at time t from nominal capacity and divide it by nominal capacity? I'm okay with the interpretation given in that referred discussion however I would like to know the discharge capacity proper definition - Is that the amount of capacity that is discharged at that particular time t? Please kindly confirm my doubts

**Reply by Akila1993** [reply]
Hi @Ramesh-Isukapalli ,

Yes, you can calculate the SOC as you mentioned, 

I use like `SoC = SoC_init - "Discharge Capacity [A.h]"/"Nominal cell capacity [A.h]`.

About the discharge capacity my understanding it can be calculated by discharge current * time, which refers to the amount of electric charge that can be released from the battery during a discharge cycle.

Hope this make sense. 

Thanks.
